### ISSUE TITLE
MapSequence.peek() should accept draw_limb and draw_grid keyword arguments

### ISSUE THREAD

**Issue by aringlis** [sunpy/sunpy#1412]
It would be nice if MapCubes could easily accept keyword arguments such as draw_limb and draw_grid. At the moment they don't seem to.

**Comment by ayshih** [comment]
Presumably just `MapCube.peek()` and not `MapCube.plot()`, since that's how it is for the corresponding for Maps.

**Comment by Cadair** [comment]
more generally than this, it would be good if you could customise mapcube plots more generally, by running a function for each map or similar.

**Comment by wafels** [comment]
Something like ...

MapCube.peek(sunpy.map.Map.draw_grid(draw_grid_props),
sunpy.map.Map.draw_limb(draw_limb_props))

?

You could also have lists of functions, where the number of elements
in the list is the same as the number of maps.

That would open up the possibility of a lot of fine-grained control on
the mapcube animation.

**Comment by Cadair** [comment]
I was thinking something more like this:

```
def my_plot(fig, ax, map):
    map.draw_grid()
    ax.plot(0,0)

mymapcube.plot(plot_func=my_plot)
```

**Comment by Cadair** [comment]
@wafels I am not sure on the list of functions thing, you might as well expand the map cube at that point.

**Comment by wtbarnes** [comment]
This is resolved by #1423 right? Presumably you can just pass in a custom function that draws the grid, limb to `MapSequenceAnimator` via `peek`?

**Comment by dstansby** [comment]
Assuming this is fixed, feel free to re-open if I've missed something.
